pkgs/top-level/python-packages.nix

```diff
@@ -5097,6 +5114,29 @@ in {
     };
   };
 
+  keyrings-alt = buildPythonPackage rec {
+    name = "${pname}-${version}";
```

Comment:
The `name` attribute is added by `buildPython*` and should therefore be removed.